### PR TITLE
Added ability to set custom Resultset class returned by find()

### PR DESCRIPTION
**Pull request by ashpumpkin**
Thank you for wanting to contribute to Phalcon.

Before you submit a Pull Request, please bear in mind that PRs must be made on a
version branch (eg. `3.0.x`). `master` marks the latest stable release so PRs
targeting `master` will be **rejected**.

Please also check the following points are being met:
- [x] Make sure that the PR passes in Travis CI to make the process more efficient.
- [x] Only use tabs for indentation.
- [x] if needed, rebase to the proper branch before submitting your pull request.
    If it doesn't merge cleanly with master you may be asked to rebase your changes.
- [x] Don't put submodule updates in your pull request unless they are to landed commits.
- [x] Add tests relevant to the fixed bug or new feature. See our testing guide for
    more information.
- [x] Phalcon 2 is written in Zephir, please do not submit commits that modify C generated
    files directly or those whose functionality/fixes are implemented in the C
    programming language
- [x] Remove any change to ext/kernel / *.zep.c / *.zep.h files before submitting the PR

### CODE REVIEW

File: tests/unit/Mvc/Model/ResultsetClassTest.php [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+{
+    /**
+     * Checks if resultset class Simple is returned when getResultsetClass() mmethod is not defined
+     *
+     * @author Eugene Smirnov <ashpumpkin@gmail.com>
+     */
+    public function testDefaultResultsetClass()
+    {
+        $this->specify(
+            'Find() method should return custom resultset if getResultsetClass() method is presented in model',
```
Comment (Green-Cat) [inline tests/unit/Mvc/Model/ResultsetClassTest.php]: 'Find() method should return the default resultset simple class if getResultsetClass() method is not presented in the model',

Comment (ashpumpkin) [inline tests/unit/Mvc/Model/ResultsetClassTest.php]: @Green-Cat ty, fixed it

**Review by sergeyklay — APPROVED** [review]
Looks good to me.

CC: @andresgutierrez

**Comment by sergeyklay** [comment]
@ashpumpkin Could you please rebase

**Comment by ashpumpkin** [comment]
@sergeyklay done

**Comment by sergeyklay** [comment]
Thanks

**Pull request merged**